src/main/java/dev/jbang/cli/BaseBuildCommand.java

```diff
@@ -137,161 +126,7 @@ static Source build(ScriptSource src, RunContext ctx) throws IOException {
 		return result;
 	}
 
-	// build with javac and then jar... todo: split up in more testable chunks
-	static private IntegrationResult buildJar(ScriptSource src, RunContext ctx, File tmpJarDir, File outjar,
```

Comment:
Why is this refactor necessary? (see also below)

Reply:
i pushed commit that "undoes" most of it.